docs/_docs/focus/convert.md

```diff
@@ -57,35 +62,33 @@ The following mapping is assuming you have all amortized cost rows and only comm
 | CommitmentDiscountType     | BenefitId                                           | If BenefitId contains "/microsoft.capacity/" (case-insensitive), `Reservation`; if contains "/microsoft.billingbenefits/", `Savings Plan`; otherwise, null            |
 | ConsumedQuantity           | Quantity                                            | If ChargeType == "Usage", then Quantity; otherwise, null                                                                                                              |
 | ConsumedUnit               | UnitOfMeasure                                       | If ChargeType == "Usage", then map using [Pricing units data file](../../_reporting/data/README.md#-pricing-units); otherwise, null                                   |
-| ContractedCost             | UnitPrice * Quantity                                | Map UnitOfMeasure using [Pricing units data file](../../_reporting/data/README.md#-pricing-units) and divide Quantity by the PricingBlockSize                         |
+| ContractedCost             | UnitPrice * Quantity / x_PricingBlockSize (lookup)  | Map UnitOfMeasure to x_PricingBlockSize using [Pricing units data file](../../_reporting/data/README.md#-pricing-units)                                               |
```

Comment:
`x_PricingBlockSize` is the column name in `FOCUS` dataset, but inside the reference data CSV file, it is the column name is `PricingBlockSize`.

Reply:
Given the mapping doesn't happen in this column, what do you think about only calling out the division here and leaving the mapping for the x_PricingBlockSize column?
```suggestion
| ContractedCost             | UnitPrice * Quantity / x_PricingBlockSize | Note that x_PricingBlockSize requires a mapping. See column notes for details. |
```

Reply:
Sounds reasonable.